<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>685</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>685</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/Resources/calificar-nota.png</normaloff>:/Resources/calificar-nota.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="outCuadro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-style:italic; color:#32989a;&quot;&gt;Promedio de notas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>20</y>
      <width>131</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Notas</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(180, 211, 102);</string>
          </property>
          <property name="text">
           <string>Nota 1:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="inNota1">
          <property name="maximum">
           <number>130</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(100, 222, 108);</string>
          </property>
          <property name="text">
           <string>Nota 2:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="inNota2">
          <property name="maximum">
           <number>130</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(91, 176, 196);</string>
          </property>
          <property name="text">
           <string>Nota 3:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="inNota3">
          <property name="maximum">
           <number>130</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdPromedio">
       <property name="text">
        <string>Promedio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionPromediar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionPromediar"/>
  </widget>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resources/disco-flexible.png</normaloff>:/Resources/disco-flexible.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPromediar">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resources/calculos.png</normaloff>:/Resources/calculos.png</iconset>
   </property>
   <property name="text">
    <string>Promediar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resources/add-file.png</normaloff>:/Resources/add-file.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPromediar</sender>
   <signal>triggered()</signal>
   <receiver>cmdPromedio</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
